dependencies.toml

```diff
@@ -659,11 +657,11 @@ javadocs = "https://www.javadoc.io/doc/ch.qos.logback/logback-classic/1.2.11/"
 [libraries.micrometer-core]
 module = "io.micrometer:micrometer-core"
 version.ref = "micrometer"
-javadocs = "https://www.javadoc.io/doc/io.micrometer/micrometer-core/1.9.1/"
+javadocs = "https://www.javadoc.io/doc/io.micrometer/micrometer-core/1.9.4/"
```

Comment:
Self note: It seems useful to template the link of javadocs. For example, `https://www.javadoc.io/doc/io.micrometer/micrometer-core/{version}/` could be populated into the `https://www.javadoc.io/doc/io.micrometer/micrometer-core/1.9.4/`